libraries/shared/fetcher/geth_rpc_storage_fetcher.go

```diff
@@ -19,7 +19,7 @@ import (
 
 	"github.com/ethereum/go-ethereum/rlp"
 	"github.com/ethereum/go-ethereum/statediff"
-	"github.com/makerdao/vulcanizedb/libraries/shared/storage/utils"
+	storage_pkg "github.com/makerdao/vulcanizedb/libraries/shared/storage"
```

Comment:
why alias here?

Reply:
On line #63 we use `storage` as a reference to an account's storage, so we get a naming collision. I suppose instead of using a pkg alias, I could the storage variable to be accountStorage or something:

```
for _, accountStorage := range account.Storage {
    diff, formatErr := storage_pkg.FromGethStateDiff(account, stateDiff, accountStorage)
    ...
}
```

Do you have a preference? 🤔

Reply:
Yeah I generally prefer renaming vars vs imports, but it's no big deal - just drawing from https://github.com/golang/go/wiki/CodeReviewComments#imports